library/download-manager/src/main/java/org/cru/godtools/download/manager/GodToolsDownloadManager.java

```diff
@@ -250,18 +247,18 @@ void pruneStaleTranslations() {
                     .map(Language::getCode)
                     .toArray();
 
-            // remove any translation that is no longer added to this device and hasn't been accessed within the past
-            // week
-            final Translation translation = new Translation();
-            translation.setDownloaded(false);
-            int changes = mDao.update(translation, TranslationTable.FIELD_TOOL.notIn(constants(tools))
-                    .or(TranslationTable.FIELD_LANGUAGE.notIn(constants(languages)))
-                    .and(TranslationTable.FIELD_LAST_ACCESSED.lt(new Date(System.currentTimeMillis() - WEEK_IN_MS)))
-                    .and(TranslationTable.SQL_WHERE_DOWNLOADED), TranslationTable.COLUMN_DOWNLOADED);
+//            // remove any translation that is no longer added to this device and hasn't been accessed within the past
+//            // week
+//            final Translation translation = new Translation();
+//            translation.setDownloaded(false);
+//            int changes = mDao.update(translation, TranslationTable.FIELD_TOOL.notIn(constants(tools))
+//                    .or(TranslationTable.FIELD_LANGUAGE.notIn(constants(languages)))
+//                    .and(TranslationTable.FIELD_LAST_ACCESSED.lt(new Date(System.currentTimeMillis() - WEEK_IN_MS)))
+//                    .and(TranslationTable.SQL_WHERE_DOWNLOADED), TranslationTable.COLUMN_DOWNLOADED);
```

Comment:
I left the code in but commented it out because long term I want to rework this to prune unused/stale translations when too much disk space is being used.